docs/quickstarts/ratify-quickstart-manual.md

```diff
@@ -0,0 +1,98 @@
+
+# Manual Quick Start Steps
+
+This document outlines the **manual** steps to install Ratify with Gatekeeper in admission control scenarios. Please refer to the [README.MD](../../README.md) for recommended install steps.
```

Comment:
Maybe the recommended steps should be the manual steps for production? The quick start is a great for a quick try. what do you think?